src/middlewares/errorHandler.ts

```diff
@@ -0,0 +1,11 @@
+import { Request, Response, NextFunction } from 'express'
+import BaseError from '../errors/BaseError'
+
+export default function errorHandler (error: BaseError, req: Request, res: Response, next: NextFunction): void {
+  const status = error.status || 500
+  const message = error.message || 'Something went wrong '
+  res.send({
+    status,
```

Comment:
There is no need to send status in the body.

Reply:
fixed c612b1feb30118ac3c59790b282badd72380a0de